frontend/public/metalkube/components/host/machine-cell.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Icon } from 'patternfly-react';
+import {
+  canHostAddMachine,
+  DASHES,
+  getHostMachineName,
+} from 'kubevirt-web-ui-components';
+
+import { MachineModel } from '../../models';
+import { ResourceLink } from '../utils/okdutils';
+import { referenceForModel } from '../../../module/k8s';
+
+const MachineCell = ({ host }) => {
+  const machineName = getHostMachineName(host);
+
+  const {
+    metadata: { namespace },
+  } = host;
+
+  if (machineName) {
+    return (
+      <ResourceLink
```

Comment:
shouldn't the ResourceLink include namespace?

Reply:
Fixed, thanks!